Source/WTF/wtf/WorkQueue.h

```diff
@@ -89,6 +89,8 @@ class WorkQueueBase : public ThreadSafeRefCounted<WorkQueueBase>, protected Thre
 class WTF_CAPABILITY("is current") WTF_EXPORT_PRIVATE WorkQueue : public WorkQueueBase, public SerialFunctionDispatcher {
 public:
     static WorkQueue& main();
+    static Ref<WorkQueue> protectedMain() { return main(); }
```

Comment:
I don't think we need to do this since it's a singleton. We should just suppress the warning if anything.

Reply:
Ok, I'll revert that part.